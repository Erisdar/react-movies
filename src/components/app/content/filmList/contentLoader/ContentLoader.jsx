import React from 'react';
import './contentLoader.css';

export default () => (<div className="content__loader"></div>);