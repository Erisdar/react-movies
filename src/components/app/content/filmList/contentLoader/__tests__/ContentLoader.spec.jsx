import { shallow } from 'enzyme';
import React from 'react';
import ContentLoader from '../ContentLoader';

describe('Search component', () => {
    it('should be render correctly', () => {
        const component = shallow(<ContentLoader />);
        expect(component).toMatchSnapshot();
    });
});