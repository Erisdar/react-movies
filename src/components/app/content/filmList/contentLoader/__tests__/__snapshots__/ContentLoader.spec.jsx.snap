// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Search component should be render correctly 1`] = `
<div
  className="content__loader"
/>
`;
